tests/Cargo.toml

```diff
@@ -139,7 +138,7 @@ test_sst_importer = { path = "../components/test_sst_importer" }
 test_raftstore = { path = "../components/test_raftstore" }
 byteorder = "1.2"
 serde_json = "1.0"
-tokio = { version = "0.2", features = ["rt-core"] }
+tokio = { version = "0.2", features = ["rt-threaded"] }
```

Comment:
Why changed? Will it make test slower?

Reply:
I removed futures-cpupool and we need to use a new thread executor using future 0.3.